<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LirchQtInterface</class>
 <widget class="QMainWindow" name="LirchQtInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Lirch @LIRCH_VERSION_STRING@</string>
  </property>
  <widget class="QWidget" name="mainWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QVBoxLayout" name="mainLayout">
      <item>
       <widget class="QSplitter" name="chatLayout">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QTabWidget" name="chatTabWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <property name="tabsClosable">
          <bool>false</bool>
         </property>
         <property name="movable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="fileTransfersTab">
          <attribute name="title">
           <string>File Transfers</string>
          </attribute>
          <layout class="QVBoxLayout" name="fileTransfersLayout">
           <item>
            <widget class="QLabel" name="all_transfers_label">
             <property name="text">
              <string>All Transfers</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QProgressBar" name="all_progress">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="value">
              <number>100</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="downloads_label">
             <property name="text">
              <string>Downloads</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QProgressBar" name="downloads_progress">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="value">
              <number>100</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="uploads_label">
             <property name="text">
              <string>Uploads</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QProgressBar" name="uploads_progress">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="value">
              <number>100</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="file_transfer_note">
             <property name="text">
              <string>Note: File Transfer is a forthcoming feature.</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QListView" name="chatUserList">
         <property name="minimumSize">
          <size>
           <width>160</width>
           <height>400</height>
          </size>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="msgLayout">
        <item>
         <widget class="QLineEdit" name="msgTextBox">
          <property name="minimumSize">
           <size>
            <width>250</width>
            <height>25</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="msgSendButton">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>Send</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="menuNew">
     <property name="title">
      <string>&amp;New</string>
     </property>
     <addaction name="actionNewChannel"/>
     <addaction name="actionNewTransfer"/>
    </widget>
    <addaction name="menuNew"/>
    <addaction name="actionConnect"/>
    <addaction name="separator"/>
    <addaction name="actionOpenLog"/>
    <addaction name="actionSaveLog"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionWizard"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <widget class="QMenu" name="menuViewTab">
     <property name="title">
      <string>&amp;Tabs</string>
     </property>
     <addaction name="actionViewTransfers"/>
     <addaction name="separator"/>
    </widget>
    <addaction name="menuViewTab"/>
    <addaction name="actionViewUserList"/>
    <addaction name="actionViewSendButton"/>
    <addaction name="separator"/>
    <addaction name="actionViewIgnored"/>
    <addaction name="actionViewTimestamps"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionEditNick"/>
    <addaction name="separator"/>
    <addaction name="actionEditIgnored"/>
    <addaction name="actionEditShown"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Connect</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+#</string>
   </property>
  </action>
  <action name="actionSaveLog">
   <property name="text">
    <string>&amp;Save Log</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
   <property name="shortcut">
    <string>F12</string>
   </property>
  </action>
  <action name="actionEditIgnored">
   <property name="text">
    <string>&amp;Ignore/Block</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionEditShown">
   <property name="text">
    <string>&amp;Unignore/Unblock</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="actionViewIgnored">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Ignored Messages</string>
   </property>
   <property name="toolTip">
    <string>Show/hide ignored messages</string>
   </property>
   <property name="shortcut">
    <string>Alt+I</string>
   </property>
  </action>
  <action name="actionEditNick">
   <property name="text">
    <string>&amp;Nick</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="actionOpenLog">
   <property name="text">
    <string>&amp;Open Log</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionViewUserList">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;User List</string>
   </property>
   <property name="toolTip">
    <string>Show/hide the user list</string>
   </property>
   <property name="shortcut">
    <string>Alt+U</string>
   </property>
  </action>
  <action name="actionWizard">
   <property name="text">
    <string>Run Setup &amp;Wizard</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionViewSendButton">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Send Button</string>
   </property>
   <property name="toolTip">
    <string>Show/hide the &quot;Send&quot; button</string>
   </property>
   <property name="shortcut">
    <string>Alt+S</string>
   </property>
  </action>
  <action name="actionViewTransfers">
   <property name="text">
    <string>&amp;File Transfers</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+`</string>
   </property>
  </action>
  <action name="actionViewTimestamps">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Timestamps</string>
   </property>
   <property name="toolTip">
    <string>Show/hide timestamps on each message</string>
   </property>
   <property name="shortcut">
    <string>Alt+T</string>
   </property>
  </action>
  <action name="actionNewChannel">
   <property name="text">
    <string>Private &amp;Channel</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+J</string>
   </property>
  </action>
  <action name="actionNewTransfer">
   <property name="text">
    <string>&amp;File Transfer</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>msgTextBox</tabstop>
  <tabstop>msgSendButton</tabstop>
  <tabstop>chatUserList</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>msgTextBox</sender>
   <signal>returnPressed()</signal>
   <receiver>msgSendButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>424</x>
     <y>468</y>
    </hint>
    <hint type="destinationlabel">
     <x>628</x>
     <y>468</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>LirchQtInterface</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>14</x>
     <y>30</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
